File: WSPTools/App/WSPBuilderTemplates/VBTemplates/Item/VB_WebService/NewWebService.vb
Imports System
Imports System.Web
Imports System.Web.Services
Imports System.Web.Services.Protocols


Namespace $DefaultNamespace$
    <WebService([Namespace]:="http://tempuri.org/")> _
    <WebServiceBinding(ConformsTo:=WsiProfiles.BasicProfile1_1)> _
    Public Class $rootname$
        Inherits System.Web.Services.WebService

        <WebMethod()> _
        Public Function HelloWorld() As String
            Return "Hello World"
        End Function

    End Class

End Namespace

File: WSPTools/App/WSPBuilderTemplates/Templates/Item/WebService/NewWebService.cs
using System;
using System.Web;
using System.Web.Services;
using System.Web.Services.Protocols;


namespace $DefaultNamespace$
{
    [WebService(Namespace = "http://tempuri.org/")]
    [WebServiceBinding(ConformsTo = WsiProfiles.BasicProfile1_1)]
    public class $rootname$ : System.Web.Services.WebService
    {

        public $rootname$() 
        {
            
        }

        [WebMethod]
        public string HelloWorld()
        {
            return "Hello World";
        }

    }

}